crates/cheatnet/src/runtime_extensions/forge_runtime_extension/mod.rs

```diff
@@ -508,6 +508,37 @@ impl<'a> ExtensionLogic for ForgeExtension<'a> {
                 }
                 Ok(CheatcodeHandlingResult::Handled(output))
             }
+            "print_call_entry_point" => {
```

Comment:
Probably we should move it to IO extension, it would be useful to make it reusable in cast.

Reply:
Also we could make it more generic and call it flexible_print or something like that so we can reuse it for other structures.